INSTALL.md

```diff
@@ -1,12 +1,12 @@
 # Installation
 These instructions are for installing and running the application in development mode on a development machine. There are two installation methods below, one uses [docker](https://www.docker.com/) and the other is manual. The docker installation is the recommended method.
 
-## Docker installation - website and satellite simulator (recommended)
+## Docker Installation - website and satellite simulator (recommended)
 The docker installation methods below are compatible with any operating system that is supported by [docker](https://www.docker.com/). If you are just looking to use the web app and don't plan on making any changes to the source code, then choose the `User Installation` method below. Otherwise, choose the `Developer Installation` method below. 
 
 Prior to starting, please [install docker](https://www.docker.com/get-started) for your operating system if you have not already.
 
-### User Installation
+## User Installation
```

Comment:
Lets keep this as `### User Installation` as it is one of the docker installation methods so should be a sub-category

```suggestion
### User Installation
```